arm/Microsoft.Network/bastionHosts/deploy.bicep

```
@@ -64,6 +74,8 @@ var diagnosticsLogs = [for log in logsToEnable: {
   }
 }]
 
+var var_scaleUnits = skuType == 'Basic' ? 2 : scaleUnits
```

```suggestion
var scaleUnits_var = skuType == 'Basic' ? 2 : scaleUnits
```
Just aligning with how vars are handled on other modules

---

ah thats what I get for not double-checking. I thought I remembered it being var_ . Good catch :)